frontend/src/pages/QADashboard/RecipientsWithOhsStandardFeiGoal/index.js

```diff
@@ -1,25 +1,56 @@
-/* eslint-disable no-alert */
-/* eslint-disable no-console */
 import React, {
-  useState,
+  // useState,
   useRef,
+  useContext,
 } from 'react';
 import moment from 'moment';
 import { Link } from 'react-router-dom';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faArrowLeft } from '@fortawesome/free-solid-svg-icons';
 import PropTypes from 'prop-types';
 import { Helmet } from 'react-helmet';
-import { Grid, Alert } from '@trussworks/react-uswds';
+import { Grid } from '@trussworks/react-uswds';
 import colors from '../../../colors';
 import RecipientsWithOhsStandardFeiGoalWidget from '../../../widgets/RecipientsWithOhsStandardFeiGoalWidget';
 import Drawer from '../../../components/Drawer';
 import ContentFromFeedByTag from '../../../components/ContentFromFeedByTag';
 import DrawerTriggerButton from '../../../components/DrawerTriggerButton';
+import FilterPanel from '../../../components/filter/FilterPanel';
+import FilterPanelContainer from '../../../components/filter/FilterPanelContainer';
+import useFilters from '../../../hooks/useFilters';
+import { QA_DASHBOARD_FILTER_KEY, QA_DASHBOARD_FILTER_CONFIG } from '../constants';
+import UserContext from '../../../UserContext';
 
+const ALLOWED_SUBFILTERS = [
```

Comment:
If we will have a bunch of these scattered should we have in constants file?

Reply:
they are on a per page basis